fabric-networking-api-v1/src/main/java/net/fabricmc/fabric/api/networking/v1/EntityTrackingEvents.java

```diff
@@ -72,4 +93,23 @@ public interface StopTracking {
 
 	private EntityTrackingEvents() {
 	}
+
+	//--
+
+	/**
+	 * @deprecated Use {#BEFORE_START_TRACKING} or {@link #AFTER_START_TRACKING} instead
+	 */
+	@Deprecated(forRemoval = true)
+	public static final Event<BeforeStartTracking> START_TRACKING = BEFORE_START_TRACKING;
```

Comment:
I would prob just leave this a tottaly seperate event, and keep it the same as it was before.

Reply:
Little clarification, do you mean:
1. Keep the original Event as it was but with deprecation while also adding the specific before and after variants 
OR
2. Keep the original Event but add the after-variant as the only addition

Reply:
Think it's no. 2; start tracking and after start tracking events.

Reply:
Hi, sorry. I was more thinking about 1:

Keep the current event untouched, no need to try and be clever about ensuring backwards compatibility just duplicate it. You would then end up with:

- BEFORE_START_TRACKING
- START_TRACKING (existing, deprecated)
- AFTER_START_TRACKING

Reply:
I did ensure the code was backward compatible but made the desired adjustment to the PR.